src/client/sandbox/node/window.js

```diff
@@ -10,7 +11,7 @@ export default class WindowSandbox extends SandboxBase {
     constructor (sandbox) {
         super(sandbox);
 
-        this.UNCAUGHT_JS_ERROR = 'uncaughtJSError';
+        this.UNCAUGHT_JS_ERROR_EVENT = 'uncaughtJSError';
```

Comment:
I'm starting to wondering if it was a good idea to keep constants on instance level. On one hand we'll be able replace values for testing purposes, on other hand we will need to make instance methods which may be static but uses this constants. I'd stick with the static constants, e.g.:
```js
WindowSandbox.UNCAUGHT_JS_ERROR_EVENT = 'uncaughtJSError';
```